packages/codegen-ui/lib/utils/form-to-component/helpers/map-element-children.ts

```diff
@@ -0,0 +1,60 @@
+/*
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+import {
+  FormDefinitionElement,
+  FormDefinitionRadioGroupFieldElement,
+  FormDefinitionSelectFieldElement,
+  StudioComponentChild,
+} from '../../../types';
+
+type MapElementChildrenReturnValue = { children: StudioComponentChild[] };
+
+function mapOptions(
```

Comment:
should this just return the component children list?

Reply:
I made it an object on purpose because when we handle relationship options later, we will be handling `bindingProperties` and probably returning something like `modelsToImport` alongside the `children`.